Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -155,14 +155,14 @@ function Main() {
 	else
 		_entry=$(cat /etc/default/grub.d/50-cloudimg-settings.cfg | grep 'rootdelay=')
 		if [ -n "$_entry" ]; then
-			sed -i -e "s/rootdelay=300/rootdelay=300 resume=$sw_uuid/g" /etc/default/grub.d/50-cloudimg-settings.cfg
+			sed -i -e "s/rootdelay=300/rootdelay=300 log_buf_len=200M resume=$sw_uuid/g" /etc/default/grub.d/50-cloudimg-settings.cfg
```

Comment:
please add some comments or log for adding the boot kernel parameter

Reply:
Fixed in 622bd33e7b08b2878c5003d009a6eee15b4cf4a2